app/src/main/java/com/example/mystore/ui/shared/RatingBar.kt

```diff
@@ -0,0 +1,40 @@
+package com.example.mystore.ui.shared
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Star
+import androidx.compose.material.icons.outlined.Star
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import com.example.mystore.ui.theme.LightGray
+import com.example.mystore.ui.theme.Purple40
+import kotlin.math.ceil
+
+
+@Composable
+fun RatingBar(
+    modifier: Modifier = Modifier,
+    rating: Float = 0.0f,
+    stars: Int = 5,
```

Comment:
This 5 can be a `static const val` constant in a `Constants` file.

Edit: I just saw you even already have one there.